packages/core-mobile/app/contexts/DeeplinkContext/DeeplinkContext.tsx

```diff
@@ -143,7 +164,11 @@ export const DeeplinkContextProvider = ({
       }
     }
 
-    checkInitialUrl()
+    checkInitialUrl().catch(reason => {
+      Logger.error(
+        `[packages/core-mobile/app/contexts/DeeplinkContext/DeeplinkContext.tsx][checkInitialUrl]${reason}`
```

Comment:
maybe declare the file path as const in the file?

Reply:
@neven-s are you just trying to give a "tag" to the logs in terminal? if so doing something like [DeeplinkContextTag] should be good enough

Reply:
[Shorten error log tags](https://github.com/ava-labs/core-mobile/pull/1659/commits/c294b918c0220a1c70121afcda28fc69ede2de9d)